CONTRIBUTING.md

```diff
@@ -21,42 +21,35 @@
 
 ## Did you find a bug?
 
-The Arrow project uses JIRA as a bug tracker.  To report a bug, you'll have
-to first create an account on the
-[Apache Foundation JIRA](https://issues.apache.org/jira/).  The JIRA server
-hosts bugs and issues for multiple Apache projects.  The JIRA project name
-for Arrow is "ARROW".
-
-To be assigned to an issue, ask an Arrow JIRA admin to go to
-[Arrow Roles](https://issues.apache.org/jira/plugins/servlet/project-config/ARROW/roles),
-click "Add users to a role," and add you to the "Contributor" role.  Most
-committers are authorized to do this; if you're a committer and aren't
-able to load that project admin page, have someone else add you to the
-necessary role.
-
-Before you create a new bug entry, we recommend you first
-[search](https://issues.apache.org/jira/projects/ARROW/issues/ARROW-5140?filter=allopenissues)
-among existing Arrow issues.
-
-When you create a new JIRA entry, please don't forget to fill the "Component"
-field.  Arrow has many subcomponents and this helps triaging and filtering
-tremendously.  Also, we conventionally prefix the issue title with the component
-name in brackets, such as "[C++] Crash in Array::Frobnicate()", so as to make
-lists more easy to navigate, and we'd be grateful if you did the same.
+The Arrow project uses GitHub as a bug tracker.  To report a bug, sign in to
+your GitHub account and select **Bug Report** in the
+[GitHub issues](https://github.com/apache/arrow/issues/new/choose).
```

Comment:
"in the github issues" sounds incomplete. Maybe the "github issues form" or "github issues selector"?

Alternatively, we could also point people to https://github.com/apache/arrow/issues and direct them to click on "New issue" ?

Reply:
Agree, would choose the second, pointing to issues and then clicking on "New issue".